### PR TITLE
[device_info]first unit test for native iOS, running with XCode (not CLI) only, with coverage

### PR DESCRIPTION
**Pull request by truongsinh**
Staring point for https://github.com/flutter/flutter/issues/10141

screen shot 2019-01-05 at 07 57 12

### CODE REVIEW

**Comment by xster** [comment]
Maybe @amirh to route?

**Comment by truongsinh** [comment]
recover from accidental delete

**Comment by truongsinh** [comment]
FYI @collinjackson @mravn-google @szakarias @tvolkert

File: packages/device_info/ios/Tests/unit_test.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
+//
+//  Created by TruongSinh Tran-Nguyen on 2019-01-05.
+//  Copyright © 2019 The Chromium Authors. All rights reserved.
```
Comment (collinjackson) [inline packages/device_info/ios/Tests/unit_test.swift]: I think this should be changed to the standard Flutter copyright block

Comment (truongsinh) [reply packages/device_info/ios/Tests/unit_test.swift]: Thanks, it's done in the latest commit

**Comment by truongsinh** [comment]
pinging @collinjackson after resolving conflict

**Comment by collinjackson** [comment]
Thanks for the contribution! Curious if anyone else is using this style of unit testing in their apps and how effective it is at catching bugs/regressions. For first-party plugins, it seems like we might want to start with broad integration testing coverage before we invest in platform-specific unit tests.

**Comment by stuartmorgan** [comment]
Thanks for the submission! However, we’re no longer accepting non-critical PRs for this plugin, as we hope in time to be able to transition users to the corresponding plugin in [the Flutter Community Plus Plugins repository](https://plus.fluttercommunity.dev/). We encourage you to submit your PR there.

We apologize for the long delay in getting back to this PR. We’re in the process of overhauling our PR triage system to respond much more quickly, as well as working through the backlog.

**Pull request closed without merging**